package.json

```diff
@@ -64,6 +64,7 @@
         "eslint-webpack-plugin": "^4.2.0",
         "file-loader": "^6.2.0",
         "node-fetch": "^2.7.0",
+        "nodemon": "^3.1.9",
```

Comment:
For `nodemon` to work, don't you need to have this as the parent process of the application?

According to the [`nodemon` readme](https://github.com/remy/nodemon?tab=readme-ov-file#usage), it sounds like you'd need to do `nodemon ./dist/server.js` at the very least for it to start monitoring the application directory.

Reply:
nodemon is used for the `npm run watch` script